app/components/FFXIV/ItemDetailsTable.tsx

```
@@ -0,0 +1,208 @@
+      filtered = filtered.filter((item) => {
+        const itemName = item.itemName.toLowerCase()
+        return itemName.includes(query)
+      })
+    }
+
+    // Apply sorting
+    return filtered.sort((a, b) => {
+      const column = columnList.find((col) => col.columnId === sortColumn)
+      if (!column) return 0
+
+      let aValue: any = column.dataAccessor
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'aValue' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)